scripts/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# script to run the python workspace_container_test.py on GitHub Actions.
+# see .github/workflows/test.yml
+
+docker-compose build
+docker-compose up &
+# ugly way to wait for all the services to be ready
+sleep 60
```

Comment:
In the future could add a loop with the workspace client calling ver() or something

Reply:
could be part of the test startup actually

Reply:
I was having trouble concocting the appropriate bash invocation to wait for the services to be ready, but I've cracked it now. 🎉

Reply:
:+1:

Reply:
although I would probably still do it in the tests so they're more self contained, but this is fine